src/braket/analog_hamiltonian_simulator/rydberg/rydberg_simulator.py

```diff
@@ -123,7 +123,7 @@ def run(
 
         # Run the solver
         # We shall adaptively change between numpy solver (RK6 method) and scipy solver
-        if len(self.configurations) <= 1000:
+        if len(self.configurations) <= 32:
```

Comment:
what's the rationale behind this change?

Reply:
Sorry that I didn't see this earlier. The rationale for this is that there is some preprocessing process for scipy solver, and that makes it run much faster than numpy solver when the system is big, but slower than numpy when the system is small. The threshold here is to set that boundary in terms of the dimension of the Hilbert space. The new benchmarking result shows that the boundary should be 32 instead of 1000.